File: entity-recognition/amazon-comprehend/AmazonComprehendEntityRecognition.java
// "text" is a string containing the input document.
// For English, set "languageCode" to "en".

AmazonComprehend client = AmazonComprehendClientBuilder
    .standard()
    .withCredentials(credentialsProvider)
    .withRegion(Regions.US_EAST_1)
    .build();

DetectEntitiesRequest request = new DetectEntitiesRequest()
    .withText(text)
    .withLanguageCode(languageCode);

DetectEntitiesResult result = client.detectEntities(request);

List<Entity> entities = result.getEntities();

File: entity-recognition/amazon-comprehend/AmazonComprehendEntityRecognition.kt
// "text" is a string containing the input document.
// For US English, set "languageCode" to "en".

val client = AmazonComprehendClientBuilder
    .standard()
    .withCredentials(credentialsProvider)
    .withRegion(Regions.US_EAST_1)
    .build()

val request = DetectEntitiesRequest()
    .withText(text)
    .withLanguageCode(languageCode)

val result = client.detectEntities(request)

val entities = result.entities
